src/components/dropdown/dropdown.ts

```diff
@@ -311,7 +317,9 @@ class BXDropdown extends ValidityMixin(HostListenerMixin(FocusMixin(LitElement))
 
     const nextItemText = nextItem.textContent;
     if (nextItemText) {
-      this._assistiveStatusText = nextItemText;
+      this._assistiveStatusText = `${nextItemText}${
+        this._inputNode && this._inputNode.value ? `, Current input is: ${this._inputNode.value}` : ''
+      }`;
```

Comment:
This string interpolation has to be translatable, similar to: https://github.com/carbon-design-system/carbon-web-components/blob/v1.12.0/src/components/pagination/pagination.ts#L115-L119

Reply:
Could you expand on what it means for the string interpolation to be translatable? Thanks Akira!

Reply:
Thank you for asking; Some helpful resources are:
https://www.infragistics.com/community/blogs/b/devtoolsguy/posts/l10n-and-i18n-best-practices
https://kazupon.github.io/vue-i18n/guide/messages.html#named-formatting